splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -146,5 +146,6 @@ private void configureHandler() throws Exception {
                         .httpMethod("POST"))
             // Log after a successful send.
             .log("Response ${body}");
+            .to(kafka(kafkaReturnTopic).brokers(kafkaBrokers));
```

Comment:
I'd suggest to have this as a separate "direct" route, connected with `direct` component.

Also there is semicolon in the line above.

Reply:
Ah, ok, thank you.